src/styles.css

```diff
@@ -979,11 +973,13 @@
   gap: 1rem;
 }
 
-.cio-quiz .cio-results-redo-and-share-button-group {
+.cio-quiz .cio-results-number-and-share-button-group {
```

Comment:
I think we can use some bottom margin here.
![image](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/2954126/993d8eb5-c0cb-40b0-97bf-17c16f6f4855)

Reply:
[8e8523e](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/148/commits/8e8523eabe4324902949ad899b57d8cbc62d04d7)